rainbow/runners/airflow/dag/rainbow_dags.py

```diff
@@ -47,12 +47,16 @@ def register_dags(configs_path):
             for pipeline in config['pipelines']:
                 pipeline_name = pipeline['pipeline']
 
-                default_args = {
+                default_args = {k: v for k, v in pipeline.items()}
+
+                override_args = {
                     'owner': config['owner'],
                     'start_date': datetime.combine(pipeline['start_date'], datetime.min.time()),
                     'depends_on_past': False,
```

Comment:
I think we need to take `depends_on_past` that user passes into consideration:

```suggestion
                    'depends_on_past': default_args['depends_on_past'] if 'depends_on_past' in default_args else False
```
And we can move string value `'depends_on_past'` tp a constant in the class named `__DEPENDS_ON_PAST` instead of the literal 3 times as in my example above.